migrations/sql/V2019.02.21.9.59__add_address_columns_to_party.sql

```diff
@@ -0,0 +1,54 @@
+ALTER TABLE party ADD COLUMN suite_no       varchar(5);
+ALTER TABLE party ADD COLUMN address_line_1 varchar(50);
+ALTER TABLE party ADD COLUMN address_line_2 varchar(50);
+ALTER TABLE party ADD COLUMN city           varchar(50);
+ALTER TABLE party ADD COLUMN province_code  varchar(2);
+ALTER TABLE party ADD COLUMN postal_code    varchar(6);
+
+COMMENT ON TABLE party IS 'Party references an individual or an organization that does business with EMPR, and any relevant contact information, such as address.';
+
+CREATE TABLE IF NOT EXISTS province_code (
+    province_code    character varying(2)                   NOT NULL,
+    description      character varying(50)                  NOT NULL,
+    display_order    integer                                NOT NULL,
```

Comment:
We could make this nullable... that way if a code is marked as "not active" we don't have to assign it a display value?

Reply:
Sure, that makes sense.